openhands/runtime/client/runtime.py

```diff
@@ -152,19 +164,18 @@ def __init__(
                 extra_deps=self.config.sandbox.runtime_extra_deps,
             )
         self.container = self._init_container(
-            self.sandbox_workspace_dir,
-            mount_dir=self.config.workspace_mount_path,
+            sandbox_workspace_dir=self.config.workspace_mount_path_in_sandbox,  # e.g. /workspace
```

Comment:
this seems like a potentially breaking change--are you sure you want mount path and not the old option?

Reply:
The "old" option was pointing towards the class _property_ named `sandbox_workspace_dir`, which references the same attribute. 
Imo using here the actual self.config makes the _mounting_ purpose much clearer than the `sandbox_workspace_dir` property. It certainly had me confused and cost me time.

Reply:
That property can probably be removed, iirc it was an artefact of some "minimal" refactoring.

Reply:
Yeh, I'll remove it from the runtime classes, it is a bit confusing with little benefit.

Reply:
By design we default the host (workspace_mount_path) to workspace_base, thus in most cases they end up "the same".
But with 2 diff file systems we _can_ have 2 diff mount points, thus the extra "workspace_mount_path_in_sandbox ".